src/sidebar/templates/search-input.html

```diff
@@ -1,16 +1,17 @@
 <form class="simple-search-form"
       name="searchForm"
+      role="search"
       ng-class="!vm.query && 'simple-search-inactive'">
   <input class="simple-search-input"
-         type="text"
+         type="search"
          name="query"
          placeholder="{{vm.loading && 'Loading' || 'Search'}}…"
          ng-disabled="vm.loading"
          ng-class="vm.inputClasses()"/>
   <button type="button" class="simple-search-icon top-bar__btn" ng-hide="vm.loading">
     <i class="h-icon-search"></i>
   </button>
-  <button type="button" class="simple-search-icon btn btn-clean" ng-show="vm.loading" disabled>
+  <button type="button" class="simple-search-icon btn btn-clean" aria-label="Search" ng-show="vm.loading" disabled>
```

Comment:
This "button" is in fact the loading spinner. It shouldn't be a `<button>` at all. I don't know how best to present a loading indicator with ARIA. Can you look into this?

Reply:
Ah, I see that now - so this `aria-labe`l should actually go on the `button` in line 10. 

Though you bring up a good point - I'm not sure how to indicate loading search results to a screenreader. Will look into that as well.